t/helper/test-quote.c

```diff
@@ -0,0 +1,30 @@
+#include "test-tool.h"
+#include "quote.h"
+#include "strbuf.h"
+#include "string.h"
+
+int cmd__quote_buf_pretty(int argc, const char **argv)
+{
+	struct strbuf buf_payload = STRBUF_INIT;
+	char* nullString = 0;
+	char empty[5] = {'\0'};
+	
```

Comment:
so argv[0] is the name of the test "quote_buf_pretty".  and argv[1] is the string to test.

we'll crash if they omit a test string.  that is, if they just type "test-tool quote_buf_pretty".

Reply:
so maybe have something like:

    if (!argv[1])
        die("missing value....");